posthog/api/survey.py

```diff
@@ -60,6 +60,8 @@ class SurveySerializer(serializers.ModelSerializer):
     created_by = UserBasicSerializer(read_only=True)
     conditions = serializers.SerializerMethodField(method_name="get_conditions", read_only=True)
     feature_flag_keys = serializers.SerializerMethodField()
+    # NB this is enforced in the UI too
+    iteration_count = serializers.IntegerField(required=False, allow_null=True, max_value=500, min_value=0)
```

Comment:
nitpick, 500 can be a const and reused across the 3 places (and 2 python files)

Reply:
yeah, i think there's probably a pass over this file to make it more DRF-y
e.g. often we have a base serializer and then vary it for different use cases, or we could split out the validation from the update call